src/miner_util.erl

```diff
@@ -87,26 +87,34 @@ mark(Module, Mark) ->
 metadata_fun() ->
     try
         Map = blockchain_worker:signed_metadata_fun(),
-        case miner_lora:position() of
-            %% GPS location that's adequately close to the asserted
-            %% location
-            {ok, _} ->
-                Map#{<<"gps_fix_quality">> => <<"good_fix">>};
-            %% the assert location is too far from the fix
-            {ok, bad_assert, _} ->
-                Map#{<<"gps_fix_quality">> => <<"bad_assert">>};
-            %% no gps fix
-            {error, no_fix} ->
-                Map#{<<"gps_fix_quality">> => <<"no_fix">>};
-            %% no location asserted somehow
-            {error, not_asserted} ->
-                Map#{<<"gps_fix_quality">> => <<"not_asserted">>};
-            %% got nonsense or a hopefully transient error, return nothing
+        case application:get_env(miner, mode, gateway) of
+            validator ->
```

Comment:
Since we don't use the gps information at all, can we just always send the 'release version'?

Reply:
this is a good idea, but we'd need to change it to read lsb-release on the gateways